lib/rubocop/cop/rspec/empty_hook.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      #  Checks for empty before and after hooks.
+      #
+      # @example
+      #   # bad
+      #   before {}
+      #   after do; end
+      #   before(:all) do
+      #   end
+      #   all(:all) { nil }
+      #
+      #   # good
+      #   before { create_users }
+      #   after do; cleanup_users; end
+      #   before(:all) do
+      #     create_feed
+      #   end
+      #   all(:all) { cleanup_feed }
+      class EmptyHook < Cop
+        MSG = 'Empty hook detected.'
+
+        def_node_matcher :empty_hook?, <<~PATTERN
+          (block
+            $(send nil? {:before :after}
+              (sym {:each :all :suite})?)
```

Comment:
There are a few aliases you're missing here. Luckily, they are conveniently defined [here](https://github.com/rubocop-hq/rubocop-rspec/blob/master/lib/rubocop/rspec/language.rb#L101).